expertise/service/expertise.py

```diff
@@ -464,23 +463,34 @@ def get_expertise_results(self, user_id, job_id, delete_on_get=False):
             ## TODO: change it to Job not found
             raise openreview.OpenReviewException(f"Scores not found - status: {status} | description: {description}")
         else:
-            # Search for scores files (only non-sparse scores)
             file_dir, metadata_dir = self._get_score_and_metadata_dir(search_dir)
             self.logger.info(f"Retrieving scores from {search_dir}")
             ret_list = []
-            with open(file_dir, 'r') as csv_file:
-                data_reader = reader(csv_file)
-                for row in data_reader:
-                    # For single paper retrieval, filter out scores against the dummy submission
-                    if row[0] == 'dummy':
-                        continue
-
-                    ret_list.append({
-                        'submission': row[0],
-                        'user': row[1],
-                        'score': float(row[2])
-                    })
-            result['results'] = ret_list
+            if 'submission_group' not in config.keys():
+                with open(file_dir, 'r') as csv_file:
+                    data_reader = reader(csv_file)
+                    for row in data_reader:
+                        # For single paper retrieval, filter out scores against the dummy submission
+                        if row[0] == 'dummy':
+                            continue
+
+                        ret_list.append({
+                            'submission': row[0],
+                            'user': row[1],
+                            'score': float(row[2])
+                        })
+                result['results'] = ret_list
+            else:
+                # If submission group, group under different keys
```

Comment:
why different keys?

Reply:
What I mean by this comment is not creating tuples of `(submission, user, score)`, create tuples of `(match_member, submission_member, score)` so these keys have to be different to be explicit on what scores are being returned.